concrete/src/Marketplace/Marketplace.php

```diff
@@ -186,9 +186,18 @@ public static function downloadRemoteFile($file)
      */
     public static function checkPackageUpdates()
     {
+        $marketplace = static::getInstance();
+        $skipPackages = $marketplace->config->get('concrete.updates.skip_packages');
+        if ($skipPackages === true) {
+            return;
+        }
+        $skipPackages = preg_split('/\s+/', (string) $skipPackages, -1, PREG_SPLIT_NO_EMPTY);
```

Comment:
We could just use an array instead of a string for this

Reply:
Done in b37652d4987d34483b5af19f93713cfe505c0c1a